packages/loader/container-loader/src/container.ts

```diff
@@ -810,6 +810,7 @@ export class Container
 		validateDriverCompatibility(
 			maybeDriverCompatDetails.ILayerCompatDetails,
 			(error) => {} /* disposeFn */, // There is nothing to dispose here, so just ignore the error.
+			subLogger,
```

Comment:
Is this the right logger?  Comparing with `this.mc.logger` used in the other spot.  Just calling it out to make sure.

Reply:
Eh probably fine.  Your tests cover the logger namespaces so it seems that you've thought it through e2e